openproblems/data/tabula_muris_senis.py

```diff
@@ -73,20 +58,31 @@ def load_tabula_muris_senis(test=False, method_list=None, organ_list=None):
     and droplet-fat anndata sets. (no facs-fat dataset available)
     """
 
-    # df containing figshare links, method of collection, and organ for each
-    # tabula muris dataset
-    url_df = pd.read_csv(
-        os.path.join(
-            os.path.dirname(__file__),
-            "tabula_muris_senis_data_objects",
-            "tabula_muris_senis_data_objects.csv",
-        ),
-        header=0,
-    )
+    if not method_list:
+        raise ValueError("`method_list' argument not provided")
+    unknown_methods = set(method_list) - set(["facs", "droplet"])
+    if unknown_methods:
+        raise ValueError(
+            f"Unknown methods provided in `method_list': {','.join(unknown_methods)}. "
+            "Known methods are `facs' and `droplet'"
+        )
+    organ_list = [x.lower() for x in organ_list]
```

Comment:
If you `.lowe()` organs, you should do the same for methods

Reply:
For methods, I think "Smart-seq2" is spelled with a capital letter in the mapping dictionary atm.

Reply:
I convert from CELLxGENE names (Smart-seq2, 10x 3' v2) to short paper code names (facs, droplet)